src/Nethermind/Nethermind.sln.DotSettings

```diff
@@ -1,4 +0,0 @@
-﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
-	<s:Boolean x:Key="/Default/UserDictionary/Words/=merkleizer/@EntryIndexedValue">True</s:Boolean>
-	<s:Boolean x:Key="/Default/UserDictionary/Words/=Precompiles/@EntryIndexedValue">True</s:Boolean>
-	<s:Boolean x:Key="/Default/UserDictionary/Words/=timestamper/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
No my custom dictionary!

Reply:
How could we build solution-level dictionary though?

Reply:
Consider the following approach(dictionary mentioned also there): https://github.com/NethermindEth/dotnet-libp2p/pull/146/files

Reply:
@flcl42 can you add it to this repo?